wallet/substrates/wallet_wire_transceiver.go

```diff
@@ -0,0 +1,51 @@
+package substrates
+
+import (
+	"github.com/bsv-blockchain/go-sdk/wallet"
+	"github.com/bsv-blockchain/go-sdk/wallet/serializer"
+)
+
+// WalletWireTransceiver implements wallet.Interface
+// A way to make remote calls to a wallet over a wallet wire.
+type WalletWireTransceiver struct {
+	Wire WalletWire
+}
+
+func NewWalletWireTransceiver(processor *WalletWireProcessor) *WalletWireTransceiver {
+	return &WalletWireTransceiver{Wire: processor}
+}
+
+func (t *WalletWireTransceiver) Transmit(call Call, originator string, params []byte) ([]byte, error) {
```

Comment:
I made a little reasearch and Transmit is a private method of WalletWireTranceiver. So you don't need to expose it.

Reply:
Cool!